### PR TITLE
Decrease min group name to 3 characters

### PR DESCRIPTION
**Pull request by hmstepanek**
CNN and other user groups of H have 3 character acronyms and would like to use their acronym as their group name. This enables them to do that by changing the minimum character limit of group names from 4 to 3.

### CODE REVIEW

**Comment by ajpeddakotla** [comment]
Thanks for this @hmstepanek!

**Comment by lyzadanger** [comment]
@hmstepanek Sorry for my terse review! I'm buried and wanted to make sure the API docs didn't get overlooked...

File: tests/h/models/groups_test.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def test_init_sets_given_attributes():
 def test_with_short_name():
     """Should raise ValueError if name shorter than 4 characters."""
```
Comment (lyzadanger) [inline tests/h/models/groups_test.py]: This code comment is no longer valid :P

**Comment by lyzadanger** [comment]
Nice use of `fixup` commit, @hmstepanek ! I'll try to get a chance to review this in more depth this afternoon.

**Comment by hmstepanek** [comment]
@lyzadanger I only found one spot in the api docs (the new-group) that needed to be updated. Let me know if I missed something!

**Comment by lyzadanger** [comment]
@hmstepanek I'd like to get either @seanh or @robertknight to give their blessing on this, simply because they have a longer history and may have insight on any possible reason that changing the min-length could be problematic (apologies if this conversation has already been had!).

**Comment by robertknight** [comment]
I'm not aware of any reason why a three-character group name should be a problem.

**Pull request merged**